### DISCUSSION TITLE
Allow disabling of gap between display edge and window edge

### DISCUSSION THREAD

**Discussion by gregnostic** [rxhanson/Rectangle#292] [Ideas]
The "Gaps between windows" feature creates gaps on all sides of the windows, even when the window shares an edge with the display. These spaces don't necessarily benefit from the gaps the same way that the spaces between windows do. It would be nice if this could be a configurable option, either as a checkbox—simply turn the gaps on or off entirely—or as a separate slider, so that the gaps around display edges could be increased or decreased independent of the gaps between windows.

**Comment by rxhanson** [comment]
Sorry for the delayed response. This idea makes sense to me, but I would only be inclined to have this go in as a terminal command toggle that either enables or disables screen edge gaps. I don't have the bandwidth for it at the moment, but I don't mind providing an entry point for interested developers.
